app/views/admin/poll/results/_results_by_booth.html.erb

```diff
@@ -0,0 +1,20 @@
+<table class="margin" id="booth_assignment_results">
```

Comment:
I'm missing an `<h3>` at the beginning, maybe "Final Recount & Result by Booth" ? 

It changes without much context from one table to another:
![screen shot 2017-10-19 at 00 26 26](https://user-images.githubusercontent.com/983242/31745706-31553510-b464-11e7-9e5f-0accc5b146e9.jpg)